app/src/main/java/org/sagebionetworks/research/mpower/research/DataSourceManager.kt

```diff
@@ -50,6 +50,19 @@ class DataSourceManager {
         fun installedGroup(forIdentifier: String): ActivityGroup? {
             return installedGroups.firstOrNull { it.identifier == forIdentifier }
         }
+
+        @JvmStatic
+        fun defaultEngagementGroups(): Set<Set<String>> {
+            return setOf(
+                setOf("gr_BR_AD", "gr_DT_T"),
```

Comment:
@syoung-smallwisdom are these the correct random engagement groups?  I just went off of the motivation survey rules.

Reply:
No. The defaults are defined in `StudyBurstConfiguration`.

```swift
        self.engagementDataGroups = [
            ["gr_SC_DB","gr_SC_CS"],
            ["gr_BR_AD","gr_BR_II"],
            ["gr_ST_T","gr_ST_F"],
            ["gr_DT_F","gr_DT_T"]]
```

So that would be:

```java
        return setOf(
            setOf("gr_SC_DB","gr_SC_CS"),
            setOf("gr_BR_AD","gr_BR_II"),
            setOf("gr_ST_T","gr_ST_F"),
            setOf("gr_DT_F","gr_DT_T"));
```